src/main/java/com/sigma/cinematicketpro/controller/handler/DefaultFilterExceptionHandler.java

```diff
@@ -0,0 +1,52 @@
+package com.sigma.cinematicketpro.controller.handler;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sigma.cinematicketpro.exception.ApiErrorResponse;
+import io.jsonwebtoken.JwtException;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.validation.constraints.NotNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class DefaultFilterExceptionHandler extends OncePerRequestFilter {
```

Comment:
It's necessary to answer a user in a standardized format in case of exception

Reply:
okay I see two parts of logic here:
1. doFilterInternal - then it your CustomRequestFilter
2. Exception handling that can be done in GlobalExceptionhandler

Reply:
If I got your point:
1. Yes, it would be more convinient to rename DefaultFilterExceptionHandler to JwtFilterExceptionHandler.
2. The GlobalExceptionHandler is a RestControllerAdvice so it will only intercept exceptions that've appeared in controllers, not in the filters.